.changeset/mean-guests-love.md

```diff
@@ -0,0 +1,11 @@
+---
+'@rocket.chat/meteor': minor
+---
+
+feat: *Enterprise* Add support for different transporters to connect multiple monolith instances.
+
+To use that, you can use the `TRANSPORTER` env var adding "monolith+" to the transporter value. To use NATS for example, your env var should be:
+
+```bash
+export TRANSPORTER="monolith+nats://localhost:4222"
```

Comment:
Is there a real downside to just making it use transporter just like normal?  This means if someone deploys this then wants to move to microservices they’ve got to now remove that instead of add microservices

Reply:
yeah I agree with Aaron